lib/models/environment-variables.coffee

```diff
@@ -154,7 +154,7 @@ exports.remove = (id, callback) ->
 # > false
 ####
 exports.isSystemVariable = (variable) ->
-	return /^RESIN_/.test(variable.name)
+	return /^RESIN(_.*)?|USER/.test(variable.name)
```

Comment:
This regex will also catch strings like RESINATOR, SOME_USER.

The correct regex would be: /^RESIN_|^RESIN$|^USER$/